cli/packages/util/config.go

```diff
@@ -31,9 +32,25 @@ func WriteInitalConfig(userCredentials *models.UserCredentials) error {
 		return fmt.Errorf("writeInitalConfig: unable to write config file because [err=%s]", err)
 	}
 
+	//if profiles exists
+	loggedInUser := models.LoggedInUser{
+		Email:  userCredentials.Email,
+		Domain: config.INFISICAL_URL,
+	}
+	if len(existingConfigFile.LoggedInUsers) > 0 {
+		ok := ConfigContainsEmail(existingConfigFile.LoggedInUsers, userCredentials.Email)
+		if !ok {
+			existingConfigFile.LoggedInUsers = append(existingConfigFile.LoggedInUsers, loggedInUser)
+		}
+	} else {
+		existingConfigFile.LoggedInUsers = append(existingConfigFile.LoggedInUsers, loggedInUser)
+	}
+
 	configFile := models.ConfigFile{
-		LoggedInUserEmail: userCredentials.Email,
-		VaultBackendType:  existingConfigFile.VaultBackendType,
+		LoggedInUserEmail:  userCredentials.Email,
+		LoggedInUserDomain: config.INFISICAL_URL,
```

Comment:
we set the current logged in user domain here but i don't think we update it in `LoggedInUsers` domain property  if the user email already exists. Let me know if i misunderstood

Reply:
Right, good eye! Will amend